docs/examples.rst

```diff
@@ -229,45 +229,45 @@ The following example demonstrates a few things:
     import sys
     import time
     import pprint
-    
+
     from web3.providers.eth_tester import EthereumTesterProvider
     from web3 import Web3
     from solc import compile_source
-    
+
 
     def compile_source_file(file_path):
        with open(file_path, 'r') as f:
           source = f.read()
-    
+
        return compile_source(source)
-    
-    
+
+
     def deploy_contract(w3, contract_interface):
         tx_hash = w3.eth.contract(
             abi=contract_interface['abi'],
             bytecode=contract_interface['bin']).deploy()
-    
+
         address = w3.eth.getTransactionReceipt(tx_hash)['contractAddress']
         return address
-    
-    
+
+
     w3 = Web3(EthereumTesterProvider())
-    
+
     contract_source_path = 'contract.sol'
     compiled_sol = compile_source_file('contract.sol')
-    
+
```

Comment:
For PR's like this that are growing quite large, pulling these kinds of changes out into their own small PRs is good practice.